GameData/RP-0/Avionics.cfg

```diff
@@ -921,21 +921,21 @@
 }
 
 //  Centaur astrionics.
-//  Approximates the maximum current Centaur + payload combined mass (23t + 25t).
+//  Centaur guidance provides control for both Centaur and Atlas stages
 
-@PART[RP0probeAvionics3-05m]:FOR[RP-0]
+@PART[RP0probeAvionics3-05m|RSBtankAtlasCentaur]:FOR[RP-0]
 {
 	MODULE
 	{
 		name = ModuleAvionics
-		massLimit = 45.0
+		massLimit = 600
 		toggleable = true
 		disabledkW = 0.02
 	}
 }
 
 // FIXME: Do this for other small upper stages.
-@PART[FASAAgenaProbe|RP0probeAvionics1-5m|RSBtankAgenaD|bluedog_agenaProbeCore]:FOR[RP-0]
+@PART[FASAAgenaProbe|RP0probeAvionics1-5m|RSBtankAgenaD|bluedog_agenaProbeCore|RSBtankAgenaD]:FOR[RP-0]
```

Comment:
Duplicate now

Reply:
Should the configs for the new parts be in separate entries from the existing parts? (sorry, not sure exactly what you mean @siimav)

Reply:
`RSBtankAgenaD` is twice on that same row